test/JDBC/expected/babel_assign_nchar-vu-prepare.out

```diff
@@ -0,0 +1,133 @@
+
+---------------------------
+-- UDT
+---------------------------
+CREATE TYPE nchar_type FROM NCHAR(30);
+CREATE TYPE nvarchar_type FROM NVARCHAR(30);
+CREATE TYPE char_type FROM CHAR(30);
+CREATE TYPE varchar_type FROM VARCHAR(30);
+CREATE TYPE binary_type FROM BINARY(10);
+CREATE TYPE varbinary_type FROM VARBINARY(10);
+GO
+
+
+---------------------------
+-- implicite function calls
+---------------------------
+CREATE FUNCTION babel_4803_varchar_func (@s VARCHAR(4000)) 
+RETURNS VARCHAR(4000) 
+AS 
+BEGIN 
+    RETURN @s; 
+END 
+GO
+
+CREATE FUNCTION babel_4803_nvarchar_func (@s NVARCHAR(4000))
+RETURNS NVARCHAR(4000)
+AS
+BEGIN
+    RETURN @s;
+END
+GO
+
+CREATE FUNCTION babel_4803_char_func (@s CHAR(50))
+RETURNS CHAR(50)
+AS
+BEGIN
+    RETURN @s;
+END
+GO
+
+CREATE FUNCTION babel_4803_nchar_func (@s NCHAR(50))
+RETURNS NCHAR(50)
+AS
+BEGIN
+    RETURN @s;
+END
+GO
+
+
+------------------------------------
+-- Procedures with declare statement
+------------------------------------
+CREATE PROCEDURE babel_4803_proc_nchar
```

Comment:
we need to have some tests like 

Create proc xyz (@in nchar(30))

exec xyz N',,,,'